server/src/main/java/org/spine3/server/command/CommandValidator.java

```diff
@@ -77,7 +77,7 @@ private CommandValidator() {
         result.addAll(messageViolations);
 
         final CommandContext context = command.getContext();
-        final String commandId = idToString(context.getCommandId()).trim();
```

Comment:
What if the command ID was set directly to have some spaces (by mistake)?

If we remove the call to `trim()` — and we are very confident that this is a good idea — then the text of the message and the corresponding constant name should be changed. Now it mentions `_OR_BLANK`, and with the change we do not check for that.